snippets/csharp/VS_Snippets_VBCSharp/CsCsrefQueryKeywords/CS/Join.cs

```diff
@@ -165,7 +165,7 @@ orderby prod.CategoryID
                 Console.WriteLine("   {0}:{1}", item.ProductName, item.Category);
             }
 
-            Console.WriteLine("GroupJoin3: {0} items in 1 group", totalItems, groupJoinQuery3.Count());
+            Console.WriteLine("GroupJoin3: {0} items in 1 group", totalItems);
```

Comment:
Can you explain why we're removing the `Count` from the output?

Reply:
@BillWagner , The old code seems to be not using the removed `Count`. There is no `{1}` in the first argument of `WriteLine`. I don't know if this is the intended behavior or not. ( I haven't read the code sample nor the article )